__mock__/mockDynamoDb.js

```diff
@@ -13,83 +24,90 @@ class MockDynamoDb {
         this.returnObject = null;
         this.deleteInput = null;
         this.putInput = null;
-    }
-    get(params) {
+        this.updateInputs = [];
+    };
+    MockDynamoDb.prototype.get = function (params) {
+        var _this = this;
         return {
-            promise: () => {
-                return new Promise((resolve, reject) => {
+            promise: function () {
```

Comment:
This conversion seems like its doing the same thing with a slightly different syntax.  Is there a specific reason why this change should be made

Reply:
this was autogenerated from a change to the .ts file

Reply:
ah, okay